PacketTunnelProvider/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -28,8 +28,9 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
     }
 
     @objc init() {
-        super.init(notificationCenter: NotificationCenter.default,
-                   notificationsPresenter: DefaultNotificationPresenter(),
+        super.init(notificationsPresenter: DefaultNotificationPresenter(),
+                   tunnelHealthStore: NetworkProtectionTunnelHealthStore(),
+                   controllerErrorStore: NetworkProtectionTunnelErrorStore(),
```

Comment:
Instead of passing the `notificationCenter` which isn't needed for iOS, we're now passing the tunnel health and controller error stores.

As a side note, these two should probably eventually be unified.